src/Selenium2Library/__init__.py

```diff
@@ -34,13 +34,22 @@ class Selenium2Library(
     imported into your Robot test suite (see `importing` section), and the 
     `Open Browser` keyword must be used to open a browser to the desired location.
 
-    = Locating elements =
+    **--- Note important change starting with Version 1.7.0 release ---**
+    *Locating or getting elements*
+    *Locating or specifying elements*
```

Comment:
Already corrected. It was a left over artefact of my indecision at the time. Not sure about the asterisks; can't remember.